application/auth/views.py

```diff
@@ -84,3 +85,11 @@ def reset_password(token):
         return render_template("auth/password_updated.html", form=form, token=token, user=user)
 
     return render_template("auth/reset_password.html", form=form, token=token, user=user)
+
+
+@auth_blueprint.route("/logout", methods=["POST"])
```

Comment:
Can we call it `sign_out` to match the label text?

Reply:
But we have `/login` for logging in and it will mess with the symmetry.  `/logout` feels pretty standard to me.  In that I would expect to be able to logout from most websites by typing /logout.  And it's not a URL that anyone will ever see in their browser bar - it's only posted to and then they're redirected to homepage.

Reply:
I don't think we need to scope renaming views in this ticket ... and `logout` is more consistent with our entrypoint.

Reply:
Fine